robot/Cumulus/resources/NPSP.robot

```diff
@@ -2,6 +2,7 @@
 
 Resource       cumulusci/robotframework/Salesforce.robot
 Library        DateTime
+Library        Collections
```

Comment:
Is the collections library for the lists? Where are you making use of this ?

Reply:
This is to enable use of the 'Append to List' keyword.